web/client/utils/StateUtils.js

```diff
@@ -190,39 +265,3 @@ export const updateStore = ({ rootReducer, rootEpic, reducers = {}, epics = {} }
     const epic = rootEpic || combineEpics(...wrapEpics(epics));
     (epicMiddleware || fetchMiddleware()).replaceEpic(epic);
 };
-
-/**
- * Updates a Redux store with new reducers and epics.
- * Needed by the dynamic plugins loading system, to update the application store with new reducers and epics exported by the plugins.
- *
- * If you want to add replace current reducers / epics with new ones, use updateStore instead.
- *
- * @param {object} options options to update
- * @param {object} options.reducers list of reducers to add.
- * @param {object} options.epics list of epics to add.
- * @param {object} store the store to update, if not specified, the persisted one will be used
- */
-export const augmentStore = ({ reducers = {}, epics = {} } = {}, store) => {
-    const rootReducer = fetchReducer();
-    const reducer = persistReducer((state, action) => {
-        const initialStoreKeys = Object.keys(rootReducer({}, {}));
-        const newState = {...state, ...rootReducer(state, action)};
-        return Object.keys(reducers)
-            // avoid to update the state twice
-            // if the original store contains the same reducers
-            .filter((key) => initialStoreKeys.indexOf(key) === -1)
-            .reduce((previous, current) => {
-                return {
-                    ...previous,
-                    [current]: reducers[current](previous[current], action)
-                };
-            }, newState);
-    });
-    (store || getStore()).replaceReducer(reducer);
-
-    const rootEpic = fetchEpic();
-
-    wrapEpics(epics).forEach((epic) => {
-        rootEpic.next(epic);
-    });
-};
```

Comment:
If we are removing the augmentStore we need to add a note in the migration guideline doc explaining the alternative with the new storeManager because this is a breaking change